package com.zx.scala.chapter09

object Scala1_Exception {
  def main(args: Array[String]): Unit = {

  }
  @throws
  def test()={
    throw new Exception
  }
}
